www/index.html

```diff
@@ -359,6 +359,7 @@
   <script type="text/javascript" src="lib/framework7/js/framework7.min.js"></script>
   <script type="text/javascript" src="js/init-styles.js"></script>
   <script type="text/javascript" src="js/my-app.js"></script>
+  <script type="text/javascript">init();</script>
```

Comment:
Probably! Before, attaching the various DOM events was done at the global-scope within `my-app.js`. So functionally, this PR doesn't change that, however your suggestion is probably safer. Note that the [Getting Started page on F7 docs](https://framework7.io/docs/get-started.html) basically describes using this exact process (attaching/interacting with DOM elements at the global scope, not inside an event listener, via a script tag added to the bottom of an HTML file) - so not sure what else would be considered "the typical F7 way".